src/select-histogram/types.ts

```diff
@@ -0,0 +1,16 @@
+import { HistogramSliderProps } from "../histogram-slider";
+import { TruncateType } from "../shared/dropdown-list-v2/types";
+import { DropdownAlignmentType } from "../shared/dropdown-wrapper";
+
+export interface SelectHistogramSliderProps {
+    error?: boolean | undefined;
+    alignment?: DropdownAlignmentType | undefined;
+    dropdownZIndex?: number | undefined;
+    histogramSlider: HistogramSliderProps;
```

Comment:
can pick specific props instead as not all are used or relevant

i.e.
`Pick<HistogramSliderProps, "bins" | "interval" | "renderEmptyView"> `

and move `onChange`, `rangeLabelPrefix` and `rangeLabelSuffix` to `SelectHistogramSliderProps`